superset/connectors/druid/models.py

```diff
@@ -220,13 +220,13 @@ def refresh(self, datasource_names, merge_flag, refreshAll):
                     if datatype == 'hyperUnique' or datatype == 'thetaSketch':
                         col_obj.count_distinct = True
                     # Allow sum/min/max for long or double
-                    if datatype == 'LONG' or datatype == 'DOUBLE':
+                    if datatype == 'LONG' or datatype in ('FLOAT', 'DOUBLE'):
```

Comment:
could go `col_obj.is_num()` that comes from the base column class and includes all these